clients/src/main/java/org/apache/kafka/clients/consumer/CommitFailedException.java

```diff
@@ -33,7 +33,7 @@ public CommitFailedException() {
                 "rebalanced and assigned the partitions to another member. This means that the time " +
                 "between subsequent calls to poll() was longer than the configured max.poll.interval.ms, " +
                 "which typically implies that the poll loop is spending too much time message processing. " +
-                "You can address this either by increasing the session timeout or by reducing the maximum " +
+                "You can address this either by increasing max.poll.interval.ms or by reducing the maximum " +
                 "size of batches returned in poll() with max.poll.records.");
```

Comment:
I don't think this description is completely accurate as the commit can fail when the coordinator has kicked us out of the group as well (for example, on session timeout). Though I am not sure if we have a great way of distinguishing between these cases based on the local consumer state.

Reply:
Yeah I agree. max.poll.interval.ms should be tuned more instead of session timeout given the heartbeat thread mechanism.